.github/workflows/compile-examples.yml

```diff
@@ -43,34 +46,14 @@ jobs:
           - fqbn: arduino:mbed_portenta:envie_m7
             platforms: |
               - name: arduino:mbed_portenta
+            additional-sketch-paths: |
+              - examples/PortentaH7Logger
```

Comment:
✅ All example and test sketches compile for the H7, C33 and Opta. Only exception is the `examples/PortentaH7Logger`, which is set to only compile for the H7.

https://github.com/arduino-libraries/Arduino_UnifiedStorage/actions/runs/6106529149/job/16571781505?pr=16#step:3:6012